real_time/rbsm_config.txt

```diff
@@ -2,7 +2,7 @@
 
 // RBSM Headers
 
-RBSM_MID_ENC_TICKS_LAST, 0
```

Comment:
It isn't used right now, so I am guessing we want to remove it, do you confirm ?